frontend/src/lib/components/AppEditorLink/AppEditorLink.js

```diff
@@ -23,12 +24,17 @@ export function AppEditorLink({ actionId, style, className, children }) {
             >
                 {children}
             </a>
-            {modalOpen && (
-                <ChooseURLModal
-                    actionId={actionId}
-                    dismissModal={() => setModalOpen(false)}
-                />
-            )}
+            <Modal
+                visible={modalOpen}
+                title={
+                    actionId
+                        ? 'Where do you want to edit this action?'
+                        : 'On which domain do you want to create an action?'
+                }
+                onDismiss={() => setModalOpen(false)}
```

Comment:
This isn't a prop for antd modal. I'd suggest using `footer={<Button onClick={() => setModalOpen(false)}>Close</Button>}` and `onCancel={() => setModalOpen(false)}`

Reply:
Done!